kibana-reports/.cypress/integration/04-download.spec.ts

```diff
@@ -89,22 +89,9 @@ describe('Cypress', () => {
       'include',
       '/opendistro_kibana_reports'
     );
-    cy.wait(12500);
-    cy.get('#createReportHomepageButton').click();
-
-    // enter a report name
-    cy.get('#reportSettingsName').type('Create cypress test on-demand report');
-
-    // enter a report description
-    cy.get('#reportSettingsDescription').type('Description for cypress test');
-
-    // create an on-demand report definition
-    cy.get('#createNewReportDefinition').click({ force: true });
-
     cy.wait(10000);
 
-    // visit the details page of the newly created on-demand definition
-    cy.get('#reportDefinitionDetailsLink').first().click();
```

Comment:
why remove those? Will it still actually create an on-demand report after removing those actions?

Reply:
The report definition and reports are already created in other cypress tests, so it is safe to remove